mobile/.env

```diff
@@ -0,0 +1 @@
+BASE_URL=https://localhost:8080
```

Comment:
I don't think we need a .env file for this, it's only one variable. I'd rather just specify the variable in the npm commands.

Reply:
yeah this should not have made it in the commit. I think we just forgot to delete it